src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseStatistics.hpp

```diff
@@ -0,0 +1,109 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file SinglePhaseStatistics.hpp
+ */
+
+#ifndef SRC_CORECOMPONENTS_PHYSICSSOLVERS_FLUIDFLOW_SINGLEPHASESTATISTICS_HPP_
+#define SRC_CORECOMPONENTS_PHYSICSSOLVERS_FLUIDFLOW_SINGLEPHASESTATISTICS_HPP_
+
+#include "physicsSolvers/FieldStatisticsBase.hpp"
+
+namespace geosx
+{
+
+class SinglePhaseBase;
+
+/**
+ * @class SinglePhaseStatistics
+ *
+ * Task class allowing for the computation of aggregate statistics in single-phase simulations
+ */
+class SinglePhaseStatistics : public FieldStatisticsBase< SinglePhaseBase >
+{
+public:
+
+  using Base = FieldStatisticsBase< SinglePhaseBase >;
```

Comment:
Do you need this? And do you need it public?

Reply:
I made it `private`.

Reply:
same